tests/avro/test_message_serializer.py

```diff
@@ -76,7 +78,10 @@ def test_encode_record_with_schema(self):
             message = self.ms.encode_record_with_schema(topic, basic, record)
             self.assertMessageIsSame(message, record, schema_id)
 
-    def test_encode_record_with_schema_sets_writers_cache_once(self):
+    def test_encode_r   ecord_with_schema_sets_writers_cache_once(self):
```

Comment:
MIND THE GAP

Reply:
the note specifying not to review this yet was evidently well founded... thanks for the feedback though useful.